server_addon/houdini/client/ayon_houdini/startup/otls/ayon_lop_import.hda/INDEX__SECTION

```diff
@@ -3,12 +3,25 @@ Label:        AYON Load Asset
 Path:         oplib:/ayon::Lop/lop_import::1.0?ayon::Lop/lop_import::1.0
 Icon:         opdef:/ayon::Lop/lop_import::1.0?IconImage
 Table:        Lop
-License:      
-Extra:        
-User:         
+License:
+Extra:
+User:
 Inputs:       0 to 1
 Subnet:       true
 Python:       false
 Empty:        false
 Modified:     Thu Jun  6 01:31:52 2024
 
+Operator:     ayon::lop_import::1.1
```

Comment:
Can you make this PR just save into the existing version instead of making a new one? The existing PR hasn't been merged yet so we're still able to just freely tweak it however we want without the need to actually version the HDAs.

Additionally, changes made here aren't backwards incompatible so I'd say we shouldn't need to version this regardless.

Reply:
implemented in https://github.com/BigRoy/ayon-core/pull/9/commits/c8f10a944c3761045a1d3a59bad631dd1cba0a46

Reply:
nice, looks good.